### DISCUSSION TITLE
Support for Kamvas 24 Plus?

### DISCUSSION THREAD

**Discussion by smoqi** [OpenTabletDriver/OpenTabletDriver#2576] [Q&A] [answered]
Hello

I was just wondering if there are any plans to support the Kamvas 24 Plus? I know that it's an absurdly big tablet for osu but sometimes I use it for fun when I'm not drawing.

Also another thing would be that whenever I try to join the discord through the link on the website it just says "whoops..." and doesn't let me join. I was going to ask in the discord first so yeah. It could just be specific to me and an issue with my account or pc but I just wanted some help idk.

![brave_Qg1CXOqGqt](https://user-images.githubusercontent.com/69320646/210732700-b7708db0-597f-4fb3-a70a-fe3bfdd57fe7.png)

**Comment by jamesbt365** [comment]
Wrong type of issue for adding tablet support. though what is your discord account?

**Reply by smoqi** [reply]
Smoqi#8957

**Comment by jamesbt365** [comment, marked as answer]
#2605
